<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPlugin::Internal::UserViewer_ContactUI</class>
 <widget class="QWidget" name="UserPlugin::Internal::UserViewer_ContactUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTextEdit" name="adressTextEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>ZipCode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::QButtonLineEdit" name="zipcodeLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>City</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Utils::QButtonLineEdit" name="cityLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Views::CountryComboBox" name="country">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Tel 1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="tel1LineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Tel 2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="tel2LineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Tel 3</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="tel3LineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Fax</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="faxLineEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="mailLineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Views::CountryComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">listviewplugin/countrycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::QButtonLineEdit</class>
   <extends>QLineEdit</extends>
   <header>utils/widgets/qbuttonlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
